app/views/admins/attachments/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="path-attachment"><%= link_to t('admin.attachments.title.new'), new_admins_attachment_path, class: 'path-new-attachment' %></div> 
+  <table class="table-attachment">
+    <thead>
+      <tr class="table-attachment-title test-title">
+        <th class="title-attachment">ID</th>
+        <th class="title-attachment">Name</th>
+        <th class="title-attachment">Image</th>
+        <th class="title-attachment">Links</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @attachments.each do |attachment| %>
+        <tr>
+          <td class="title-attachment"><%= attachment.id %></td>
+          <td class="title-attachment"><%= attachment.name %></td>
+          <td class="title-attachment"> 
+            <%= image_tag attachment.media_file.variant(resize: "200x200") %>
+          </td>
+          <td class="title-attachment links">
+            <%= link_to fa_icon('eye', text: t('admin.attachments.view_link')), admins_attachment_path(attachment), class: "attachment-link" %> 
+            <%= link_to fa_icon('edit', text: t('admin.attachments.edit_link')), edit_admins_attachment_path(attachment), class: "attachment-link" %> 
+            <%= link_to fa_icon('trash', text: t('admin.attachments.delete_link')), admins_attachment_path(attachment), method: :delete, class: "attachment-link" %> 
+          </td>
+        </tr>
+      <% end %>  
+    </tbody>
+  </table> 
+</div>
```

Comment:
this div is redundant or remove end div on line 1. at the end of that line you have `</div>`